Repalce `unittest.TestCase.assesrtTrue` with plain `assert`

`unittest.TestCase.assesrtTrue` should be replaced with `assert` to get more detailed test failure information.

### Example:

```python
# test.py

import unittest


class TestRunStatus(unittest.TestCase):
    def test_assert_true_1(self):
        x = "a"
        y = "b"
        self.assertTrue(x == y)

    def test_assert_true_2(self):
        x = "a"
        y = ["b", "c"]
        self.assertTrue(x in y)

    def test_assert_1(self):
        x = "a"
        y = "b"
        assert x == y

    def test_assert_2(self):
        x = "a"
        y = ["b", "c"]
        assert x in y

```

```
pytest test.py
```

```
__________________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_1 __________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_1>

    def test_assert_1(self):
        x = "a"
        y = "b"
>       assert x == y
E       AssertionError: assert 'a' == 'b'
E         - b
E         + a

self       = <test_a.TestRunStatus testMethod=test_assert_1>
x          = 'a'
y          = 'b'

test_a.py:18: AssertionError
__________________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_2 __________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_2>

    def test_assert_2(self):
        x = "a"
        y = ["b", "c"]
>       assert x in y
E       AssertionError: assert 'a' in ['b', 'c']

self       = <test_a.TestRunStatus testMethod=test_assert_2>
x          = 'a'
y          = ['b', 'c']

test_a.py:23: AssertionError
_______________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_true_1 ________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_true_1>

    def test_assert_true_1(self):
        x = "a"
        y = "b"
>       self.assertTrue(x == y)
E       AssertionError: False is not true

self       = <test_a.TestRunStatus testMethod=test_assert_true_1>
x          = 'a'
y          = 'b'

test_a.py:8: AssertionError
_______________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_true_2 ________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_true_2>

    def test_assert_true_2(self):
        x = "a"
        y = ["b", "c"]
>       self.assertTrue(x in y)
E       AssertionError: False is not true

self       = <test_a.TestRunStatus testMethod=test_assert_true_2>
x          = 'a'
y          = ['b', 'c']

test_a.py:13: AssertionError
```

- `assertTrue` only tells us `True` or `False`.
- `assert` (rewritten by pytest) tells us what's different.

--- a/tests/entities/test_run_status.py
+++ b/tests/entities/test_run_status.py
@@ -44,8 +44,8 @@ class TestRunStatus(unittest.TestCase):
             RunStatus.from_string("the IMPOSSIBLE status string")
 
     def test_is_terminated(self):
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FAILED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FINISHED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.KILLED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.SCHEDULED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.RUNNING))
+        assert (RunStatus.is_terminated(RunStatus.FAILED))
+        assert (RunStatus.is_terminated(RunStatus.FINISHED))
+        assert (RunStatus.is_terminated(RunStatus.KILLED))
+        assert not (RunStatus.is_terminated(RunStatus.SCHEDULED))
+        assert not (RunStatus.is_terminated(RunStatus.RUNNING))

--- a/tests/entities/test_run_status.py
+++ b/tests/entities/test_run_status.py
@@ -44,8 +44,8 @@ class TestRunStatus(unittest.TestCase):
             RunStatus.from_string("the IMPOSSIBLE status string")
 
     def test_is_terminated(self):
-        assert (RunStatus.is_terminated(RunStatus.FAILED))
-        assert (RunStatus.is_terminated(RunStatus.FINISHED))
-        assert (RunStatus.is_terminated(RunStatus.KILLED))
+        assert RunStatus.is_terminated(RunStatus.FAILED)
+        assert RunStatus.is_terminated(RunStatus.FINISHED)
+        assert RunStatus.is_terminated(RunStatus.KILLED)
         assert not RunStatus.is_terminated(RunStatus.SCHEDULED)
         assert not RunStatus.is_terminated(RunStatus.RUNNING)

--- a/tests/entities/test_run_status.py
+++ b/tests/entities/test_run_status.py
@@ -47,5 +47,5 @@ class TestRunStatus(unittest.TestCase):
         assert (RunStatus.is_terminated(RunStatus.FAILED))
         assert (RunStatus.is_terminated(RunStatus.FINISHED))
         assert (RunStatus.is_terminated(RunStatus.KILLED))
-        assert not (RunStatus.is_terminated(RunStatus.SCHEDULED))
-        assert not (RunStatus.is_terminated(RunStatus.RUNNING))
+        assert not RunStatus.is_terminated(RunStatus.SCHEDULED)
+        assert not RunStatus.is_terminated(RunStatus.RUNNING)

--- a/tests/store/tracking/test_file_store.py
+++ b/tests/store/tracking/test_file_store.py
@@ -153,7 +153,7 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
         fs = FileStore(self.test_root)
         for exp in fs.list_experiments():
             exp_id = exp.experiment_id
-            self.assertTrue(exp_id in self.experiments)
+            assert (exp_id in self.experiments)
             self.assertEqual(exp.name, self.exp_data[exp_id]["name"])
             self.assertEqual(exp.artifact_location, self.exp_data[exp_id]["artifact_location"])
 
@@ -478,9 +478,9 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
 
         # delete it
         fs.delete_experiment(exp_id)
-        self.assertTrue(exp_id not in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
-        self.assertTrue(exp_id in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
-        self.assertTrue(exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
+        assert (exp_id not in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
+        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
+        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
         self.assertEqual(fs.get_experiment(exp_id).lifecycle_stage, LifecycleStage.DELETED)
 
         # restore it
@@ -491,9 +491,9 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
         restored_2 = fs.get_experiment_by_name(exp_name)
         self.assertEqual(restored_2.experiment_id, exp_id)
         self.assertEqual(restored_2.name, exp_name)
-        self.assertTrue(exp_id in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
-        self.assertTrue(exp_id not in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
-        self.assertTrue(exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
+        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
+        assert (exp_id not in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
+        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
         self.assertEqual(fs.get_experiment(exp_id).lifecycle_stage, LifecycleStage.ACTIVE)
 
     def test_rename_experiment(self):

--- a/tests/store/tracking/test_file_store.py
+++ b/tests/store/tracking/test_file_store.py
@@ -153,7 +153,7 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
         fs = FileStore(self.test_root)
         for exp in fs.list_experiments():
             exp_id = exp.experiment_id
-            assert (exp_id in self.experiments)
+            assert exp_id in self.experiments
             self.assertEqual(exp.name, self.exp_data[exp_id]["name"])
             self.assertEqual(exp.artifact_location, self.exp_data[exp_id]["artifact_location"])
 
@@ -478,9 +478,9 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
 
         # delete it
         fs.delete_experiment(exp_id)
-        assert (exp_id not in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
-        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
-        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
+        assert exp_id not in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY))
+        assert exp_id in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY))
+        assert exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL))
         self.assertEqual(fs.get_experiment(exp_id).lifecycle_stage, LifecycleStage.DELETED)
 
         # restore it
@@ -491,9 +491,9 @@ class TestFileStore(unittest.TestCase, AbstractStoreTest):
         restored_2 = fs.get_experiment_by_name(exp_name)
         self.assertEqual(restored_2.experiment_id, exp_id)
         self.assertEqual(restored_2.name, exp_name)
-        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY)))
-        assert (exp_id not in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY)))
-        assert (exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL)))
+        assert exp_id in self._extract_ids(fs.list_experiments(ViewType.ACTIVE_ONLY))
+        assert exp_id not in self._extract_ids(fs.list_experiments(ViewType.DELETED_ONLY))
+        assert exp_id in self._extract_ids(fs.list_experiments(ViewType.ALL))
         self.assertEqual(fs.get_experiment(exp_id).lifecycle_stage, LifecycleStage.ACTIVE)
 
     def test_rename_experiment(self):

--- a/tests/store/tracking/test_sqlalchemy_store.py
+++ b/tests/store/tracking/test_sqlalchemy_store.py
@@ -271,8 +271,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.assertEqual(len(deleted_run_list), 2)
         for deleted_run in deleted_run_list:
             self.assertEqual(deleted_run.lifecycle_stage, entities.LifecycleStage.DELETED)
-            assert (deleted_run.experiment_id in experiment_id)
-            assert (deleted_run.run_id in run_ids)
+            assert deleted_run.experiment_id in experiment_id
+            assert deleted_run.run_id in run_ids
             with self.store.ManagedSessionMaker() as session:
                 assert self.store._get_run(session, deleted_run.run_id).deleted_time is not None
 
@@ -287,8 +287,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             self.assertEqual(restored_run.lifecycle_stage, entities.LifecycleStage.ACTIVE)
             with self.store.ManagedSessionMaker() as session:
                 assert self.store._get_run(session, restored_run.run_id).deleted_time is None
-            assert (restored_run.experiment_id in experiment_id)
-            assert (restored_run.run_id in run_ids)
+            assert restored_run.experiment_id in experiment_id
+            assert restored_run.run_id in run_ids
 
     def test_get_experiment(self):
         name = "goku"
@@ -893,7 +893,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store.log_metric(run.info.run_id, neg_inf_metric)
 
         run = self.store.get_run(run.info.run_id)
-        assert (tkey in run.data.metrics and run.data.metrics[tkey] == tval)
+        assert tkey in run.data.metrics and run.data.metrics[tkey] == tval
 
         # SQL store _get_run method returns full history of recorded metrics.
         # Should return duplicates as well
@@ -902,9 +902,9 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             sql_run_metrics = self.store._get_run(session, run.info.run_id).metrics
             self.assertEqual(5, len(sql_run_metrics))
             self.assertEqual(4, len(run.data.metrics))
-            assert (math.isnan(run.data.metrics["NaN"]))
-            assert (run.data.metrics["PosInf"] == 1.7976931348623157e308)
-            assert (run.data.metrics["NegInf"] == -1.7976931348623157e308)
+            assert math.isnan(run.data.metrics["NaN"])
+            assert run.data.metrics["PosInf"] == 1.7976931348623157e308
+            assert run.data.metrics["NegInf"] == -1.7976931348623157e308
 
     def test_log_metric_concurrent_logging_succeeds(self):
         """
@@ -1021,7 +1021,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
 
         run = self.store.get_run(run.info.run_id)
         self.assertEqual(2, len(run.data.params))
-        assert (tkey in run.data.params and run.data.params[tkey] == tval)
+        assert tkey in run.data.params and run.data.params[tkey] == tval
 
     def test_log_param_uniqueness(self):
         run = self._run_factory()
@@ -1047,7 +1047,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
 
         run = self.store.get_run(run.info.run_id)
         self.assertEqual(2, len(run.data.params))
-        assert (tkey in run.data.params and run.data.params[tkey] == tval)
+        assert tkey in run.data.params and run.data.params[tkey] == tval
 
     def test_log_null_param(self):
         run = self._run_factory()
@@ -1083,27 +1083,27 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         new_tag = entities.RunTag("tag0", "value00000")
         self.store.set_experiment_tag(exp_id, tag)
         experiment = self.store.get_experiment(exp_id)
-        assert (experiment.tags["tag0"] == "value0")
+        assert experiment.tags["tag0"] == "value0"
         # test that updating a tag works
         self.store.set_experiment_tag(exp_id, new_tag)
         experiment = self.store.get_experiment(exp_id)
-        assert (experiment.tags["tag0"] == "value00000")
+        assert experiment.tags["tag0"] == "value00000"
         # test that setting a tag on 1 experiment does not impact another experiment.
         exp_id_2 = self._experiment_factory("setExperimentTagExp2")
         experiment2 = self.store.get_experiment(exp_id_2)
-        assert (len(experiment2.tags) == 0)
+        assert len(experiment2.tags) == 0
         # setting a tag on different experiments maintains different values across experiments
         different_tag = entities.RunTag("tag0", "differentValue")
         self.store.set_experiment_tag(exp_id_2, different_tag)
         experiment = self.store.get_experiment(exp_id)
-        assert (experiment.tags["tag0"] == "value00000")
+        assert experiment.tags["tag0"] == "value00000"
         experiment2 = self.store.get_experiment(exp_id_2)
-        assert (experiment2.tags["tag0"] == "differentValue")
+        assert experiment2.tags["tag0"] == "differentValue"
         # test can set multi-line tags
         multiLineTag = entities.ExperimentTag("multiline tag", "value2\nvalue2\nvalue2")
         self.store.set_experiment_tag(exp_id, multiLineTag)
         experiment = self.store.get_experiment(exp_id)
-        assert (experiment.tags["multiline tag"] == "value2\nvalue2\nvalue2")
+        assert experiment.tags["multiline tag"] == "value2\nvalue2\nvalue2"
         # test cannot set tags that are too long
         longTag = entities.ExperimentTag("longTagKey", "a" * 5001)
         with pytest.raises(MlflowException, match="exceeded length limit of 5000"):
@@ -1133,7 +1133,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         # test can set tags that are somewhat long
         self.store.set_tag(run.info.run_id, entities.RunTag("longTagKey", "a" * 4999))
         run = self.store.get_run(run.info.run_id)
-        assert (tkey in run.data.tags and run.data.tags[tkey] == new_val)
+        assert tkey in run.data.tags and run.data.tags[tkey] == new_val
 
     def test_delete_tag(self):
         run = self._run_factory()
@@ -1146,8 +1146,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         # delete a tag and check whether it is correctly deleted.
         self.store.delete_tag(run.info.run_id, k0)
         run = self.store.get_run(run.info.run_id)
-        assert (k0 not in run.data.tags)
-        assert (k1 in run.data.tags and run.data.tags[k1] == v1)
+        assert k0 not in run.data.tags
+        assert k1 in run.data.tags and run.data.tags[k1] == v1
 
         # test that deleting a tag works correctly with multiple runs having the same tag.
         run2 = self._run_factory(config=self._get_run_configs(run.info.experiment_id))
@@ -1156,8 +1156,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store.delete_tag(run.info.run_id, k0)
         run = self.store.get_run(run.info.run_id)
         run2 = self.store.get_run(run2.info.run_id)
-        assert (k0 not in run.data.tags)
-        assert (k0 in run2.data.tags)
+        assert k0 not in run.data.tags
+        assert k0 in run2.data.tags
         # test that you cannot delete tags that don't exist.
         with pytest.raises(MlflowException, match="No tag with name"):
             self.store.delete_tag(run.info.run_id, "fakeTag")
@@ -1303,7 +1303,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.assertEqual(metric_obj.value, 34.0)
         self.assertEqual(metric_obj.timestamp, 85)
         self.assertEqual(metric_obj.step, 1)
-        assert ({("t1345", "tv44")} <= set(run.data.tags.items()))
+        assert {("t1345", "tv44")} <= set(run.data.tags.items())
 
     # Tests for Search API
     def _search(
@@ -2017,7 +2017,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store._log_metrics(run.info.run_id, metrics)
 
         run = self.store.get_run(run.info.run_id)
-        assert (tkey in run.data.metrics and run.data.metrics[tkey] == tval)
+        assert tkey in run.data.metrics and run.data.metrics[tkey] == tval
 
         # SQL store _get_run method returns full history of recorded metrics.
         # Should return duplicates as well
@@ -2026,9 +2026,9 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             sql_run_metrics = self.store._get_run(session, run.info.run_id).metrics
             self.assertEqual(5, len(sql_run_metrics))
             self.assertEqual(4, len(run.data.metrics))
-            assert (math.isnan(run.data.metrics["NaN"]))
-            assert (run.data.metrics["PosInf"] == 1.7976931348623157e308)
-            assert (run.data.metrics["NegInf"] == -1.7976931348623157e308)
+            assert math.isnan(run.data.metrics["NaN"])
+            assert run.data.metrics["PosInf"] == 1.7976931348623157e308
+            assert run.data.metrics["NegInf"] == -1.7976931348623157e308
 
     def test_log_batch_same_metric_repeated_single_req(self):
         run = self._run_factory()

--- a/tests/store/tracking/test_sqlalchemy_store.py
+++ b/tests/store/tracking/test_sqlalchemy_store.py
@@ -271,8 +271,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.assertEqual(len(deleted_run_list), 2)
         for deleted_run in deleted_run_list:
             self.assertEqual(deleted_run.lifecycle_stage, entities.LifecycleStage.DELETED)
-            self.assertTrue(deleted_run.experiment_id in experiment_id)
-            self.assertTrue(deleted_run.run_id in run_ids)
+            assert (deleted_run.experiment_id in experiment_id)
+            assert (deleted_run.run_id in run_ids)
             with self.store.ManagedSessionMaker() as session:
                 assert self.store._get_run(session, deleted_run.run_id).deleted_time is not None
 
@@ -287,8 +287,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             self.assertEqual(restored_run.lifecycle_stage, entities.LifecycleStage.ACTIVE)
             with self.store.ManagedSessionMaker() as session:
                 assert self.store._get_run(session, restored_run.run_id).deleted_time is None
-            self.assertTrue(restored_run.experiment_id in experiment_id)
-            self.assertTrue(restored_run.run_id in run_ids)
+            assert (restored_run.experiment_id in experiment_id)
+            assert (restored_run.run_id in run_ids)
 
     def test_get_experiment(self):
         name = "goku"
@@ -835,7 +835,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         with self.store.ManagedSessionMaker() as session:
             actual = session.query(models.SqlRun).filter_by(run_uuid=run.info.run_id).first()
             self.assertEqual(actual.lifecycle_stage, entities.LifecycleStage.DELETED)
-            self.assertTrue(
+            assert (
                 actual.deleted_time is not None
             )  # deleted time should be updated and thus not None anymore
 
@@ -893,7 +893,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store.log_metric(run.info.run_id, neg_inf_metric)
 
         run = self.store.get_run(run.info.run_id)
-        self.assertTrue(tkey in run.data.metrics and run.data.metrics[tkey] == tval)
+        assert (tkey in run.data.metrics and run.data.metrics[tkey] == tval)
 
         # SQL store _get_run method returns full history of recorded metrics.
         # Should return duplicates as well
@@ -902,9 +902,9 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             sql_run_metrics = self.store._get_run(session, run.info.run_id).metrics
             self.assertEqual(5, len(sql_run_metrics))
             self.assertEqual(4, len(run.data.metrics))
-            self.assertTrue(math.isnan(run.data.metrics["NaN"]))
-            self.assertTrue(run.data.metrics["PosInf"] == 1.7976931348623157e308)
-            self.assertTrue(run.data.metrics["NegInf"] == -1.7976931348623157e308)
+            assert (math.isnan(run.data.metrics["NaN"]))
+            assert (run.data.metrics["PosInf"] == 1.7976931348623157e308)
+            assert (run.data.metrics["NegInf"] == -1.7976931348623157e308)
 
     def test_log_metric_concurrent_logging_succeeds(self):
         """
@@ -1021,7 +1021,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
 
         run = self.store.get_run(run.info.run_id)
         self.assertEqual(2, len(run.data.params))
-        self.assertTrue(tkey in run.data.params and run.data.params[tkey] == tval)
+        assert (tkey in run.data.params and run.data.params[tkey] == tval)
 
     def test_log_param_uniqueness(self):
         run = self._run_factory()
@@ -1047,7 +1047,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
 
         run = self.store.get_run(run.info.run_id)
         self.assertEqual(2, len(run.data.params))
-        self.assertTrue(tkey in run.data.params and run.data.params[tkey] == tval)
+        assert (tkey in run.data.params and run.data.params[tkey] == tval)
 
     def test_log_null_param(self):
         run = self._run_factory()
@@ -1083,27 +1083,27 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         new_tag = entities.RunTag("tag0", "value00000")
         self.store.set_experiment_tag(exp_id, tag)
         experiment = self.store.get_experiment(exp_id)
-        self.assertTrue(experiment.tags["tag0"] == "value0")
+        assert (experiment.tags["tag0"] == "value0")
         # test that updating a tag works
         self.store.set_experiment_tag(exp_id, new_tag)
         experiment = self.store.get_experiment(exp_id)
-        self.assertTrue(experiment.tags["tag0"] == "value00000")
+        assert (experiment.tags["tag0"] == "value00000")
         # test that setting a tag on 1 experiment does not impact another experiment.
         exp_id_2 = self._experiment_factory("setExperimentTagExp2")
         experiment2 = self.store.get_experiment(exp_id_2)
-        self.assertTrue(len(experiment2.tags) == 0)
+        assert (len(experiment2.tags) == 0)
         # setting a tag on different experiments maintains different values across experiments
         different_tag = entities.RunTag("tag0", "differentValue")
         self.store.set_experiment_tag(exp_id_2, different_tag)
         experiment = self.store.get_experiment(exp_id)
-        self.assertTrue(experiment.tags["tag0"] == "value00000")
+        assert (experiment.tags["tag0"] == "value00000")
         experiment2 = self.store.get_experiment(exp_id_2)
-        self.assertTrue(experiment2.tags["tag0"] == "differentValue")
+        assert (experiment2.tags["tag0"] == "differentValue")
         # test can set multi-line tags
         multiLineTag = entities.ExperimentTag("multiline tag", "value2\nvalue2\nvalue2")
         self.store.set_experiment_tag(exp_id, multiLineTag)
         experiment = self.store.get_experiment(exp_id)
-        self.assertTrue(experiment.tags["multiline tag"] == "value2\nvalue2\nvalue2")
+        assert (experiment.tags["multiline tag"] == "value2\nvalue2\nvalue2")
         # test cannot set tags that are too long
         longTag = entities.ExperimentTag("longTagKey", "a" * 5001)
         with pytest.raises(MlflowException, match="exceeded length limit of 5000"):
@@ -1133,7 +1133,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         # test can set tags that are somewhat long
         self.store.set_tag(run.info.run_id, entities.RunTag("longTagKey", "a" * 4999))
         run = self.store.get_run(run.info.run_id)
-        self.assertTrue(tkey in run.data.tags and run.data.tags[tkey] == new_val)
+        assert (tkey in run.data.tags and run.data.tags[tkey] == new_val)
 
     def test_delete_tag(self):
         run = self._run_factory()
@@ -1146,8 +1146,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         # delete a tag and check whether it is correctly deleted.
         self.store.delete_tag(run.info.run_id, k0)
         run = self.store.get_run(run.info.run_id)
-        self.assertTrue(k0 not in run.data.tags)
-        self.assertTrue(k1 in run.data.tags and run.data.tags[k1] == v1)
+        assert (k0 not in run.data.tags)
+        assert (k1 in run.data.tags and run.data.tags[k1] == v1)
 
         # test that deleting a tag works correctly with multiple runs having the same tag.
         run2 = self._run_factory(config=self._get_run_configs(run.info.experiment_id))
@@ -1156,8 +1156,8 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store.delete_tag(run.info.run_id, k0)
         run = self.store.get_run(run.info.run_id)
         run2 = self.store.get_run(run2.info.run_id)
-        self.assertTrue(k0 not in run.data.tags)
-        self.assertTrue(k0 in run2.data.tags)
+        assert (k0 not in run.data.tags)
+        assert (k0 in run2.data.tags)
         # test that you cannot delete tags that don't exist.
         with pytest.raises(MlflowException, match="No tag with name"):
             self.store.delete_tag(run.info.run_id, "fakeTag")
@@ -1303,7 +1303,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.assertEqual(metric_obj.value, 34.0)
         self.assertEqual(metric_obj.timestamp, 85)
         self.assertEqual(metric_obj.step, 1)
-        self.assertTrue({("t1345", "tv44")} <= set(run.data.tags.items()))
+        assert ({("t1345", "tv44")} <= set(run.data.tags.items()))
 
     # Tests for Search API
     def _search(
@@ -2017,7 +2017,7 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
         self.store._log_metrics(run.info.run_id, metrics)
 
         run = self.store.get_run(run.info.run_id)
-        self.assertTrue(tkey in run.data.metrics and run.data.metrics[tkey] == tval)
+        assert (tkey in run.data.metrics and run.data.metrics[tkey] == tval)
 
         # SQL store _get_run method returns full history of recorded metrics.
         # Should return duplicates as well
@@ -2026,9 +2026,9 @@ class TestSqlAlchemyStore(unittest.TestCase, AbstractStoreTest):
             sql_run_metrics = self.store._get_run(session, run.info.run_id).metrics
             self.assertEqual(5, len(sql_run_metrics))
             self.assertEqual(4, len(run.data.metrics))
-            self.assertTrue(math.isnan(run.data.metrics["NaN"]))
-            self.assertTrue(run.data.metrics["PosInf"] == 1.7976931348623157e308)
-            self.assertTrue(run.data.metrics["NegInf"] == -1.7976931348623157e308)
+            assert (math.isnan(run.data.metrics["NaN"]))
+            assert (run.data.metrics["PosInf"] == 1.7976931348623157e308)
+            assert (run.data.metrics["NegInf"] == -1.7976931348623157e308)
 
     def test_log_batch_same_metric_repeated_single_req(self):
         run = self._run_factory()

